src/foam/dao/EasyDAO.js

```diff
@@ -100,38 +100,116 @@ foam.dao.DAO delegate = getInnerDAO() == null ?
 
 if ( delegate instanceof foam.dao.MDAO ) setMdao((foam.dao.MDAO)delegate);
 
+PipelinedPMDAO pipelineDAO = null;
+
+if(getPipelinedPMDAO()){
```

Comment:
FOAM coding standard is:
if ( getPipelinedPMDAO() ) {
See: https://github.com/foam-framework/foam/wiki/CodingStyle